generators/client/templates/react/src/main/webapp/app/app.tsx.ejs

```diff
@@ -65,15 +65,17 @@ export class App extends React.Component<IAppProps> {
               isSwaggerEnabled={this.props.isSwaggerEnabled}
             />
           </ErrorBoundary>
-          <div className="container-fluid view-container" id="app-view-container">
-            <Card className="jh-card">
+          <Container fluid tag="main" role="main"  className="view-container mt-3" id="app-view-container">
```

Comment:
I would prefer not to do any layout updates in this PR, if required this can be done separately. Can you do only the SCSS part in this PR

Reply:
:+1: I will keep only the minimum required changes in this PR. Shall I leave dart-sass in?

Reply:
yes please